test/e2e/federated-service.go

```diff
@@ -56,7 +56,7 @@ const (
 	FederatedDNS1123Regexp = `([a-z0-9]([-a-z0-9]*[a-z0-9])?\.)*([a-z0-9]([-a-z0-9]*[a-z0-9])?)`
 )
 
-var _ = framework.KubeDescribe("Service [Feature:Federation]", func() {
+var _ = framework.KubeDescribe("Federated Services", func() {
```

Comment:
thx
Just wanted to make sure since we dont want to be running our Before and AfterEach in kubernetes test suite and break them :)